CMakeLists.txt

```diff
@@ -5,10 +5,9 @@ project(roboteam_robothub)
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 SET(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake_modules")
 
-#set(CMAKE_AUTOMOC ON)
```

Comment:
We're linking against QT5, so there has to be QT5 things in here. Please don't disable this.

Reply:
I did not.